tzlocal/win32.py

```diff
@@ -137,7 +137,7 @@ def get_localzone():
     return _cache_tz
 
 
-def reload_localzone():
+def reload_localzone() -> None:
```

Comment:
This also returns a ZoneInfo, doesn't it?
```suggestion
def reload_localzone() -> zoneinfo.ZoneInfo:
```